bitcoin/src/crypto/sighash.rs

```diff
@@ -1793,4 +1865,29 @@ mod tests {
                 .unwrap()[..],
         );
     }
+
+    #[test]
```

Comment:
Adding feature gate here `#[cfg(feature = "rand-std")]` should allow you to remove the manifest/dependency change.